index.html

```diff
@@ -238,7 +239,7 @@
 			<dl class="idl" title="[Constructor(DOMString type, optional PointerEventInit eventInitDict)] interface PointerEvent : MouseEvent" data-merge="PointerEventInit">
 				<dt>readonly attribute long pointerId</dt>
 				<dd>
-					<p>A unique identifier for the pointer causing the event. This identifier MUST be unique from all other <a data-lt="active pointer">active pointers</a> at the time. A user agent MAY recycle previously retired values for <code>pointerId</code> from previous active pointers, if necessary.</p>
+					<p>A unique identifier for the pointer causing the event. This identifier MUST be unique from all other <a data-lt="active pointer">active pointers</a> in the <a href="https://www.w3.org/TR/html51/browsers.html#top-level-browsing-context">top-level browsing context</a> at the time. A user agent MAY recycle previously retired values for <code>pointerId</code> from previous active pointers, if necessary.</p>
 					
```

Comment:
In order to refer to another spec I believe we're supposed to add an entry to the "references" section (which we normally do with the `[[!HTML5]]` syntax.  We refer to W3C HTML5 all over the place, but not W3C HTML5.1.  I know I said it was fine to refer to the HTML 5.1 WD but I think it's probably simplest to just update this link to https://www.w3.org/TR/html5/browsers.html#top-level-browsing-context.

Alternately we could update all the references to HTML5 to point at HTML5.1 instead (this is all pretty silly IMHO - sorry).